package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "build": "lerna exec --scope @datapunt/matomo-tracker-js -- yarn build && tsc -b --clean && tsc -b packages/**/tsconfig.json && tsc -b packages/**/tsconfig.es.json",
+    "build": "lerna exec --scope @datapunt/matomo-tracker-js -- yarn build && tsc -b --clean && tsc -b packages/js/tsconfig.json && tsc -b packages/js/tsconfig.es.json",
```

Comment:
Now we only build the `js` package, we also need to build the `react` package. For me the glob works though... Do you use windows?

Reply:
Yes I use windows, I was thinking that this might be the issue.
However I'd say that `lerna exec --scope @datapunt/matomo-tracker-js` restricts compilation to `js` package anyway

Reply:
ah check, however we can discard this PR when https://github.com/Amsterdam/matomo-tracker/pull/197 is merged :)